text/0000-partial-lambda-args.md

```diff
@@ -0,0 +1,184 @@
+- Feature Name: `partial_lambda_args`
```

Comment:
Rust does not call these "lambdas"; using the accepted terminology language would better demonstrate how much effort was put into the RFC.

Reply:
[Well](https://doc.rust-lang.org/stable/rust-by-example/fn/closures.html).

Reply:
Closure is still the official term.

Reply:
I am updating it with _closures_, then.